features/step_definitions/role.js

```diff
@@ -116,7 +116,7 @@ var apiSteps = function () {
       return callback('Fixture for role ' + role + ' does not exist');
     }
 
-    index = Object.keys(this.roles[role].controllers)[0];
+    // todo : This test seams to have been wrongly adapted, controller variable is undefined
```

Comment:
hmm well, this test looks ugly anyway... I'm looking at it and try to fix it in another PR ;)